diff.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import json
+
+
+def compare_objects(obj1, obj2):
+    diff_funcs = []
+    if obj1['name'] == obj2['name']:
+        if obj1['functions'] != obj2['functions'] or obj1['coverage'] != obj2['coverage']:
+            if obj1['functions'] == obj2['functions']:
+                obj1['functions'] = []
+            if obj1['functions'] != obj2['functions']:
+                for func1 in obj1['functions']:
+                    for func2 in obj2['functions']:
+                        if func1['name'] == func2['name']:
+                            if func1['exec'] is False and func2['exec'] is True:
+                                func1['exec'] = True
+                                diff_funcs.append(func1)
+                obj1['functions'] = diff_funcs
+            list_val = []
+            for i, j in zip(obj1['coverage'], obj2['coverage']):
+                if j is not None and j is not 0 and j >= i:
+                    i = j - i
+                list_val.append(i)
```

Comment:
If `i` is 3 and `j` is 0, the result should be `0`. With your implementation it is `3`.